ci-operator/jobs/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main-presubmits.yaml

```diff
@@ -144,6 +144,78 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-serial,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build03
```

Comment:
my only question here is that i thought this field specified where the test should run, is `build03` an ibm cluster or am i misunderstanding this field?

Reply:
It is where the test runs, but that is just the pod that is setting up the cluster, this doesn't need to be on the platform that the destination of the test is for. All of our test clusters are on AWS IIRC.

Reply:
perfect, thanks for the explanation Joel